contracts/l1/Bridge.sol

```diff
@@ -40,6 +40,16 @@ contract Bridge is IBridge, VersionedInitializable {
         _;
     }
 
+    //returns the available rewards on the bridge : claimed+pending before consuming any bridge_rewards message from l2, otherwise the user loses all his rewards after the message is consumed and the _transfer reverts
```

Comment:
the comment should be changed here as rewards can't be lost

Reply:
yes indeed, we will have a rollback here, but the main purpose again is to optimize this transaction for users and save them the computation cost that they would have to pay before an obvious revert happens.